invenio_app_ils/records_relations/api.py

```diff
@@ -421,12 +421,10 @@ def remove(self, previous_rec, next_rec, relation_type):
 class IlsRecordWithRelations(IlsRecord):
     """Add relations functionalities to records."""
 
-    @property
-    def relations(self):
+    def relations(self, sort_by=None):
```

Comment:
we could go for `kwargs` instead:
```suggestion
    def relations(self, **kwargs):
```